Sources/MessageIndex.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+require_once($sourcedir . '/Subs-MessageIndex.php');
```

Comment:
I'd bring this require statement inside the MessageIndex function - there's no other route that comes this way, and leaving it outside the controller means it'll throw a 500 error if you happen to call the file directly (good for spamming host error logs)